libs/components/text-editor/src/lib/modules/text-editor/url-modal/text-editor-url-modal.component.html

```diff
@@ -49,6 +53,14 @@
             </option>
           </select>
         </sky-input-box>
+        <ng-template #onlyOpenLinksInNewWindow>
+          <div class="sky-font-deemphasized">
```

Comment:
Since this text serves as part of the description of the URL input, the `input` element should have an `aria-describedby` attribute that points to this element when it is displayed. You'll need to use `SkyIdService` to generate an ID for this `div` and bind it to `[id]` on this element and `[attr.aria-describedby]` on the URL input when applicable.

Reply:
Check out latest commit, I _think_ I did it correctly in generating the id for the div and setting the [`attr.aria-describedby]` attribute to the input element **only** if `openLinksInNewWindowOnly` is true.